content/400-pulse/250-database-setup/300-railway.mdx

```diff
@@ -8,93 +8,91 @@ toc: true
 
 </TopBlock>
 
-## Setup using the template
-
-<br />
-
-1. Deploy the [template](https://railway.app/template/pulse-pg) on Railway
-
-2. Click on the service called `restart-db-then-delete-me`.
+## Setup with template: "Prisma Pulse DB Only"
 
+1. Deploy the [template](https://railway.app/template/pulse-pg) on Railway.
+2. Click on the service called **restart-db-then-delete-me**.
 3. You will see a list of deployments under the **Deployments** tab.
-
 4. Click the most recent build's **View Logs** button.
-
-5. Click on the **Deploy Logs** tab.
-
-> If the service ran correctly, you should see a message in the logs that says "All done please restart the database" along with your `DATABASE_URL` connection string.
-
+5. Click on the **Deploy Logs** tab. If the service ran correctly, you should see a message in the logs that says:
+   ```terminal no-copy
+   All done please restart the database and delete this service.
+   Here is the DATABASE_URL
+   postgres://postgres:123456@127.0.0.1:5432/dummy
+   ```
+   > **Note**:
+   > The `DATABASE_URL` can also be found by clicking on the **Postgres** Service and navigating to the **Variables** tab, then clicking the copy icon next to `DATABASE_URL` environment variable value.
+   > ![](../images/railway-database-url.png)
 6. Copy the `DATABASE_URL` connection string and save it for later. Then restart your Railway database.
-
-   - Click on your database.
-
-   - Navigate to the Deployments tab.
-
-   - Go into the three-dots menu on the latest deployment and click the `Restart` option.
-
-7. After restarting your database. Click on the `restart-db-then-delete-me` service and navigate to the **Settings** tab.
-
-8. Scroll down to the bottom and click the red **Delete Service** button.
-
-   > Note: If you would like to use this service and the corresponding repository to create a new Prisma Pulse project. You can do so by cloning the repo from your GitHub account to your local machine.
-
-   - Once you have cloned the repository, you can run the following command:
-
-     ```bash
-
-     bash rm config-db.ts
-
-     ```
-
-   - Then remove the script `start: ts-node config-db.ts` from the `scripts` object in the `package.json` file.
-
-     > This is to prevent the script from running every time you push up to the repo associated with the `restart-db-then-delete-me` service.
+   1. Click on your database.
+   2. Navigate to the Deployments tab.
+   3. Go into the three-dots menu on the latest deployment and click the **Restart** option.
+7. After restarting your database, click on the **restart-db-then-delete-me** service and navigate to the **Settings** tab.
+   - Scroll down to the bottom and click the red **Delete service** button.
+8. Congrats 🎉! You should have a Prisma Pulse-compatible PostgreSQL database running on [railway.app](https://railway.app/).
+
+## Setup with template: "Prisma Pulse DB & App"
```

Comment:
```suggestion
## Setup with template: "Prisma Pulse DB & App"

This section provides step-by-step instructions for setting up a Pulse starter project with a Pulse-ready PostgreSQL database using the Railway template called: [Prisma Pulse DB & App](https://railway.app/template/pulse-starter). 
	
The template consists of three [services](https://docs.railway.app/reference/services):
- **Pulse-Starter**: A Typescript app with a basic setup for using Prisma Pulse. You can find the repo for it [on GitHub](https://github.com/prisma/pulse-starter/tree/main).
- **Postgres**: A PostgreSQL database that you'll be able to use with Prisma Pulse after going through the setup instructions.
- **restart-and-delete-me**: This service contains a [script](https://github.com/prisma/pulse-railway-pg-config/blob/main/config-db.ts) that's executed once and configures the PostgreSQL database to make it usable with Prisma Pulse. Once the script has been executed, the instructions will tell you to delete this service since it won't be needed any more.
```